AutonomousDemo/VehicleMotion.cs

```diff
@@ -19,7 +19,7 @@ public VehicleMotion(double currentSpeed)
             this.CurrentSpeed = currentSpeed;
         }
 
-        public Velocity PressGasPedal(double meters, int seconds)
+        public Velocity Drive(double meters, int seconds)
```

Comment:
This still throws me. Please provide me with a 1-2 sentence English description of what this function does (or should do).